clang/lib/DPCT/DPCT.cpp

```diff
@@ -808,7 +808,9 @@ int runDPCT(int argc, const char **argv) {
     }
     StartPos = StartPos + StartStr.length();
     EndPos = SourceCode.find_last_of('\n', EndPos);
-    QueryAPIMappingStr += SourceCode.substr(StartPos, EndPos - StartPos + 1);
+    QueryAPIMappingStr +=
+        "\033[32m" + SourceCode.substr(StartPos, EndPos - StartPos + 1).str() +
```

Comment:
suggest to define a Macor/constExpr to replace the "033[32m" with a right name to improve readability.

Reply:
fixed the next comment and this problem is not exist.